WordPress/src/main/java/org/wordpress/android/ui/posts/InsertMediaDialog.java

```diff
@@ -73,15 +76,18 @@ public String toString() {
     private SeekBar mNumColumnsSeekBar;
 
     private InsertMediaCallback mCallback;
+    private SiteModel mSite;
     private GalleryType mGalleryType;
     private InsertType mInsertType;
     private int mNumColumns;
 
 
-    public static InsertMediaDialog newInstance(@NonNull InsertMediaCallback callback) {
+    public static InsertMediaDialog newInstance(@NonNull InsertMediaCallback callback, @NonNull SiteModel site) {
         InsertMediaDialog dialog = new InsertMediaDialog();
         dialog.setStyle(AppCompatDialogFragment.STYLE_NORMAL, R.style.Theme_AppCompat_Light_Dialog);
         dialog.setCallback(callback);
+        Bundle args = new Bundle();
+        args.putSerializable(WordPress.SITE, site);
         return dialog;
```

Comment:
@daniloercoli That was really dumb of me. Corrected in 0655416, thanks for catching it.